src/request/index.js

```diff
@@ -0,0 +1,60 @@
+var http = require('http')
```

Comment:
This is what you're after and it's already there: https://github.com/mulesoft/js-client-oauth2/pull/29/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12. If you're not sure how this works, I recommend you read the super short spec on the `browser` field.